src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,84 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+
+import { actions as CurrentTodoActions } from '../../features/currentTodo';
+
+type Props = {
+  selectedUser: User | null,
+  changeUserId(todoId: number | null): void,
+  loadUser(): void,
+  resetUser(): void,
+};
+
+export const TodoModal: React.FC<Props> = (
+  {
+    selectedUser,
+    changeUserId,
+    loadUser,
+    resetUser,
+  },
+) => {
+  useMemo(() => {
+    loadUser();
+  }, []);
```

Comment:
useMemo is used for something else in React, you can take a look [here](https://react.dev/reference/react/useMemo)
```suggestion
  useEffect(() => {
    loadUser();
  }, []);
```